src/sentry/integrations/jira/search.py

```diff
@@ -57,84 +54,22 @@ def get(self, request, organization, integration_id):
                 'value': i['key']
             } for i in resp.get('issues', [])])
 
+        # TODO: remove jira_url in follow up deploy.
```

Comment:
Absolutely nothing. I plan on removing it from the front end in a follow up change. I didn't want to do it now as it could create release boundary issues.